eng/tools/generator/README.md

```diff
@@ -1,19 +1,55 @@
 # Generator
 
-This is a command line tool for generating new releases for `github.com/Azure/azure-sdk-for-go`.
+This is a command line tool for generating new releases and managing automation for `github.com/Azure/azure-sdk-for-go`.
+
+## Overview
+
+The generator tool provides several commands to support the Azure SDK for Go development lifecycle:
+
+- **Issue Management**: Parse GitHub release request issues into configuration
+- **Release Generation**: Generate new SDK releases from TypeSpec or Swagger specifications  
+- **Automation**: Process batch SDK generation for CI/CD pipelines
+- **Refresh**: Regenerate all existing SDK packages
+- **Templates**: Generate package templates and scaffolding
 
 ## Commands
 
-This CLI tool provides 2 commands now: `automation`, `issue`.
+This CLI tool provides the following commands:
+
+### The `issue` command
+
+The `issue` command fetches release request issues from `github.com/Azure/sdk-release-request/issues` and parses them into configuration that other commands consume. The configuration outputs to stdout.
+
+**Usage:**
+```bash
+generator issue [flags]
+```
 
-### The `issue` command and the configuration file
+**Flags:**
+- `-t, --token`: Personal access token for GitHub authentication
```

Comment:
Shelling out to using the `gh` CLI would be nice here - then I don't have to authenticate _twice_, you can just rely on the GitHub CLI to do the work.

You can get back JSON from the github cli as well, which I find useful sometimes:

```bash
gh issue list --json author,comments,number
```

It'll also simplify your tools docs and flags.